editor/src/components/canvas/ui-jsx-canvas.test-utils.tsx

```diff
@@ -183,15 +163,12 @@ export function renderCanvasReturnResultAndError(
       editedTextElement: null,
       mountCount: 0,
       walkDOM: false,
-      imports: imports,
-      topLevelElementsIncludingScenes: topLevelElements,
-      jsxFactoryFunction: jsxFactoryFunction,
+      imports_KILLME_ONLY_USED_FOR_CSS: imports,
```

Comment:
Do we really need to name this like this? I'd much prefer a `FIXME` comment against it so that I can find it.